### PR TITLE
fix: allowing boot and graphql endpoints to bots

### PR DESCRIPTION
**Pull request by ilasw**
Removing `/boot` and `/graphql` from disallowed paths. 

But maybe we want do the the reverse at some point disabling only the crucial ones and allowing the rest [like gitlab do](https://gitlab.com/gitlab-org/gitlab/-/merge_requests/39520).

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]
Yeah agreed on the notion of rather disallowing only specifics.
But for now this is perfect.

**Comment by capJavert** [comment]
We are doing this in order to try and fix SEO?

**Comment by ilasw** [comment]
@capJavert Yes, [this is the ticket](https://dailydotdev.atlassian.net/jira/software/projects/MI/boards/31?selectedIssue=MI-531)

File: src/routes/index.ts [hunk 1]
```diff
@@ -52,6 +52,8 @@ export default async function (fastify: FastifyInstance): Promise<void> {
   fastify.get('/robots.txt', (req, res) => {
     return res.type('text/plain').send(`User-agent: *
 Allow: /devcards/
+Allow: /graphql/
```
Comment (omBratteng) [inline src/routes/index.ts:55]: IIRC I think it's only `/graphql`
```suggestion
Allow: /graphql
```

Comment (ilasw) [reply src/routes/index.ts]: Yes, thanks!

**Review by capJavert — APPROVED** [review]

**Review by nensidosari — APPROVED** [review]

**Pull request merged**